src/tbp/monty/frameworks/experiments/pretraining_experiments.py

```diff
@@ -139,16 +164,16 @@ def pre_episode(self):
         if self.show_sensor_output:
             self.live_plotter.initialize_online_plotting()
 
+    def pre_epoch(self):
+        super().pre_epoch()
+
     def post_epoch(self):
```

Comment:
Is there any reason why we're not calling `super().post_epoch()` somewhere in this method? Or are we completely overriding its behaviour?